src/components/ebay-spark-line/component.js

```diff
@@ -0,0 +1,37 @@
+export default {
+    onCreate(input) {
```

Comment:
A lot of what's taking place in here should be moved out into function(s) and be called from here to make things easier to read, reference, and understand.